r/tools/nixlibs.R

```diff
@@ -736,11 +738,6 @@ with_cloud_support <- function(env_var_list) {
       cat("**** ", start_msg, " support ", msg, "; building with ", off_flags, "\n")
     }
 
-    # Check the features
```

Comment:
Why delete this comment? Is it no longer accurate or useful?

Reply:
I explained it in the commit message which seems redundant now as it doesn't show up unless you check ^^

The use of cmake here is correct in the fail fast way and matches the actual build, which the test compilation would not be. 

On the other hand the test compilation does not require cmake and is used to get the binary so we would need cmake just to check for the deps which is an unnecessary dependency.

So I removed the comment because there is no change necessary!

Reply:
I'm not sure I follow what you're saying here as to why it's redundant (or not used)? Could you explain a bit more for me?